sdk/communication/azure-communication-sms/src/samples/java/com/azure/communication/sms/samples/quickstart/ReadmeSamples.java

```diff
@@ -72,24 +55,52 @@ public void createSmsClienttWithConnectionString() {
             .buildClient();
     }
 
-    /**
-     * Sample code for creating a sync Communication Identity Client using AAD authentication.
-     *
-     * @return the Communication Identity Client.
-     */
-    public SmsClient createCommunicationIdentityClientWithAAD() {
-        // You can find your endpoint and access key from your resource in the Azure Portal
-        String endpoint = "https://<RESOURCE_NAME>.communication.azure.com";
+    public void sendMessageToOneRecipient(SmsClient smsClient) {
+        //Send an sms to only one phone number
+        String to = "<to-phone-number>";
 
-        // Create an HttpClient builder of your choice and customize it
-        HttpClient httpClient = new NettyAsyncHttpClientBuilder().build();
+        // to enable a delivery report to the Azure Event Grid
+        SmsSendOptions options = new SmsSendOptions();
+        options.setDeliveryReportEnabled(true);
+        //addionaly you can ad a tag you wish to identify the messages for this tag.
```

Comment:
Remove the comment on line 62 & 65, We should keep these to a minimum

Reply:
wouldn't be a little more help for the user to have this comments?.